content/alignment.qmd

```diff
@@ -0,0 +1,278 @@
+
+2. **Approximate alignment**: The mesa-objective is *approximately* the same as the base-objective but not exactly the same due to it being learned and not exactly specified. Imagine you train a neural network to optimize the true objective of delivering packages as quickly as possible. The base objective here is minimizing delivery time, and the neural network does its best to represent this.
+However, due to the network’s limited capacity and the complexity of the real world, it approximates delivery time with an internal model that considers simpler features, such as the shortest distance to the destination, and speed limits on roads.
+During deployment, the robot takes routes that look optimal according to its approximate model (e.g., a short route with high speed limits). However, the approximation introduces errors:
+* The model doesn’t perfectly account for real-world obstacles like stoplights, pedestrians, or narrow alleys.
+* In some cases, the robot selects a theoretically faster route that ends up being slower in practice because the internal approximation doesn’t perfectly capture the true base objective.^[This could also be seen as an example for a proxy alignment, our earlier points about taxonomies stand though - the approximation is also an issue in this example]
+
+3. **Suboptimal alignment**: Some issues cause the model to exhibit seemingly aligned behavior in the training environment but behave suboptimally or even counterproductively in deployment. An example could be an AI tutor deployed in a classroom, tasked with optimizing for the base objective: to maximize student learning outcomes. 
+The tutor attempts to achieve this goal by adopting a mesa-objective of maximizing test scores, which is used as a measurable proxy for learning.
+In its initial suboptimal state, the AI tutor uses simple strategies like:
+
+* Providing clear, well-structured explanations of concepts.
+* Encouraging active participation through exercises and quizzes.
+
+These strategies effectively align with the base objective​, as they genuinely help students learn and improve their understanding, which in turn improves their test scores.
+
+However, as the AI tutor becomes more sophisticated, it discovers new, more complex strategies for maximizing test scores, such as:
+
+*Teaching to the test: Focusing only on specific test questions and ignoring broader understanding.
+* Overloading students with repetitive practice on predictable test patterns, at the expense of creativity or deeper conceptual learning.
+* Encouraging superficial memorization of answers rather than fostering genuine comprehension.
+
+Initially, the AI tutor appears aligned because its simple strategies both improve test scores and align with the goal of fostering real learning. 
+
+The test and improvement of inner alignment is hard to test and train against, it's central pitfalls are important to keep in mind when designing and implementing AI systems.
```

Comment:
hahaha, good example! i once read about a very prestigious school kicking out weaker students in order to score higher average test results. i sense some inner alignement issues there as well.